web/src/state_data.ts

```diff
@@ -84,9 +84,9 @@ export const realm_schema = z.object({
     realm_avatar_changes_disabled: z.boolean(),
     realm_bot_domain: z.string(),
     realm_can_access_all_users_group: z.number(),
+    realm_can_create_public_channel_group: z.number(),
     realm_create_multiuse_invite_group: z.number(),
     realm_create_private_stream_policy: z.number(),
-    realm_create_public_stream_policy: z.number(),
```

Comment:
This field is still passed from the server but we do not use it in webapp, so there is no error on removing the field from here.